microprofile/tests/server/pom.xml

```diff
@@ -71,4 +71,25 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+    <build>
```

Comment:
This change does not seem to be required for the described problem. This is `tests/server`, not TCK

Reply:
I don't remember the context of this. Probably it was necessary at some point when this PR had other changes. It is removed now.